test/unit/facet_example_fetcher_test.rb

```diff
@@ -43,6 +43,26 @@ def stub_index(name)
     index
   end
 
+  context "#prepare_response" do
+    should "map an empty response" do
+      fetcher = FacetExampleFetcher.new(@index, {}, {}, @builder)
+
+      response = fetcher.send(:prepare_response, [], [])
+
+      assert_equal response, {}
+    end
+
+    should "map a response to facets without fields" do
+      fetcher = FacetExampleFetcher.new(@index, {}, {}, @builder)
+      slugs = ['yolo']
+      response_list = [{ 'hits' => { 'total' => 1, 'hits' => [ { 'slug' => 'yolo' }]}}]
```

Comment:
Do the responses from elasticsearch actually look like this?  Specifically, I wouldn't expect the 'slug' key to exist.  I'd expect it to look more like:

`[{ 'hits' => { 'total' => 1, 'hits' => [ { '_id' => 'yolo' }]}}]`